pkgs/by-name/ba/bant/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  buildBazelPackage,
+  fetchFromGitHub,
+  bazel_6,
+  jdk,
+  git,
+}:
+
+let
+  system = stdenv.hostPlatform.system;
+  registry = stdenv.mkDerivation {
+    name = "bazel-central-registry";
+    dontBuild = true;
+    src = fetchFromGitHub {
+      owner = "bazelbuild";
+      repo = "bazel-central-registry";
+      rev = "1c729c2775715fd98f0f948a512eb173213250da";
+      hash = "sha256-1iaDDM8/v8KCOUjPgLUtZVta7rMzwlIK//cCoLUrb/s=";
+    };
+    installPhase = ''
+       cp -r $src $out
+    '';
+  };
```

Comment:
i think you don't need to copy twice (fetchFromGithub already puts it into the store):
```
registry = fetchFromGitHub {
      owner = "bazelbuild";
      repo = "bazel-central-registry";
      rev = "1c729c2775715fd98f0f948a512eb173213250da";
      hash = "sha256-1iaDDM8/v8KCOUjPgLUtZVta7rMzwlIK//cCoLUrb/s=";
  };
```

Reply:
Thanks!